flake: TestServer_X11_EvictionLRU

```
    x11_test.go:247: 
        	Error Trace:	/home/runner/work/coder/coder/testutil/chan.go:37
        	            				/home/runner/work/coder/coder/agent/agentssh/x11_test.go:247
        	Error:      	RequireReceive: context expired
        	Test:       	TestServer_X11_EvictionLRU
    t.go:106: 2025-08-07 08:43:26.592 [debu]  closing server
    x11_test.go:241: 
        	Error Trace:	/home/runner/work/coder/coder/agent/agentssh/x11_test.go:241
        	            				/opt/hostedtoolcache/go/1.24.6/x64/src/runtime/asm_amd64.s:1700
        	Error:      	Received unexpected error:
        	            	context deadline exceeded
        	Test:       	TestServer_X11_EvictionLRU
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x50 pc=0x181bf0a]

goroutine 2347 [running]:
github.com/coder/coder/v2/agent/agentssh_test.TestServer_X11_EvictionLRU.func2()
	/home/runner/work/coder/coder/agent/agentssh/x11_test.go:242 +0xea
created by github.com/coder/coder/v2/agent/agentssh_test.TestServer_X11_EvictionLRU in goroutine 70
	/home/runner/work/coder/coder/agent/agentssh/x11_test.go:239 +0xe5e
```

seen here: https://github.com/coder/coder/runs/47577449329

--- a/agent/agentssh/x11_test.go
+++ b/agent/agentssh/x11_test.go
@@ -135,7 +135,7 @@ func TestServer_X11_EvictionLRU(t *testing.T) {
 		t.Skip("X11 forwarding is only supported on Linux")
 	}
 
-	ctx := testutil.Context(t, testutil.WaitLong)
+	ctx := testutil.Context(t, testutil.WaitSuperLong)
 	logger := testutil.Logger(t)
 	fs := afero.NewMemMapFs()
 
@@ -238,7 +238,9 @@ func TestServer_X11_EvictionLRU(t *testing.T) {
 	payload := "hello world"
 	go func() {
 		conn, err := inproc.Dial(ctx, testutil.NewAddr("tcp", fmt.Sprintf("localhost:%d", agentssh.X11StartPort+agentssh.X11DefaultDisplayOffset)))
-		assert.NoError(t, err)
+		if !assert.NoError(t, err) {
+			return
+		}
 		_, err = conn.Write([]byte(payload))
 		assert.NoError(t, err)
 		_ = conn.Close()

